src/main/java/com/example/cat/Cat.java

```diff
@@ -0,0 +1,4 @@
+package com.example.cat;
+
+public record Cat(String NAME, int AGE) {
```

Comment:
Klassen Attribute sollten immer klein geschrieben sein. 
```suggestion
public record Cat(String name, int age) {
```